### ISSUE TITLE
 Not compatible with tensorflow 1.3?

### ISSUE THREAD

**Issue by nsknsl** [lanpa/tensorboard-pytorch#17]
After I installed tensorflow 1.3 ,tensorboard can be used in terminal 
But after I installed tensorboard-pytorch, it can not work:
```
Traceback (most recent call last):
  File "/usr/local/bin/tensorboard", line 7, in <module>
    from tensorboard.main import main
ImportError: No module named 'tensorboard.main'
```

**Comment by lanpa** [comment]
Which command triggered the error message?
I am using tensorboard-pytorch v0.7 from pip, TF v1.3 and it looks fine.
Please also provide the output of `pip freeze`, thanks.

**Comment by pnavarre** [comment]
I think the problem is that they moved Tensorboard to a dependency of TF 1.3 (https://pypi.python.org/pypi/tensorboard). "tensorflow-pytorch" and the original "tensorboard" both use the same name. With both modules installed I get the following error:

> from tensorboard import SummaryWriter
Traceback (most recent call last):
>
>  File "<ipython-input-1-57c9fdbf7904>", line 1, in <module>
    from tensorboard import SummaryWriter
>
>ImportError: cannot import name 'SummaryWriter'

**Comment by lanpa** [comment]
@pnavarre The link you provided is not an official package. The official package comes with TF1.3 is https://pypi.python.org/pypi/tensorflow-tensorboard .

**Comment by pnavarre** [comment]
Sorry, my bad. I did install the official (tensorflow-tensorboad) and got the error above.

**Comment by lanpa** [comment]
@pnavarre  please post the output of  `pip freeze`. BTW do you use anaconda environment or not?

**Comment by lanpa** [comment]
@nsknsl , @pnavarre  I just uploaded https://pypi.python.org/pypi/tensorboardX 

`from tensorboardX import SummaryWriter`

This one should be fine :)

**Comment by pnavarre** [comment]
Thanks!

**Comment by lanpa** [comment]
@Ascrypto Can you paste a more detailed error message?

**Comment by Ascrypto** [comment]
@lanpa Thank you for responding back. 

The error is: 
Traceback (most recent call last):
  File "./run.py", line 29, in <module>
    from tensorboardX import SummaryWriter
ModuleNotFoundError: No module named 'tensorboardX'

Within a jupyter notebook, it works fine, but outside the notebook (when imported in .py files), it doesn't.  

 
I have the packages installed: 
 tensorboard==2.0.1
tensorboard-plugin-wit==1.8.0
tensorboardX==2.4.1
tensorflow-estimator==2.2.0

**Comment by lanpa** [comment]
@Ascrypto Would it work if you run `from tensorboardX import SummaryWriter` in an interactive python environment? Which virtual environment manager are you using? Can you post the whole package list of the virtual environment you are using?

**Comment by Ascrypto** [comment]
I appreciate your help. The problem is fixed at the moment. 

I have tried these commands to install tensorboardX inside a venv: 
- sudo pip -H install tensorboardX
- pip install tensorboardX 
- sudo pip install tensorboardX
- added the package manually from: settings --> python interpreter --> +  (in PyCharm) 

None of this worked when running **from tensorboardX import SummaryWriter** in an interactive python environment. 

When I installed it using: 
- python -m pip install tensorboardX 
- sudo python -m pip install tensorboardX 

It is now working in both python and sudo python interactive environments.
